src/multi-verso/MultiVerso/Path.lean

```diff
@@ -24,10 +24,15 @@ namespace Path
 /--
 Retrieves a string that can be used as a link.
 -/
-public def link (path : Path) (htmlId : Option String := none) : String :=
-  "/" ++ String.join (path.toList.map (· ++ "/")) ++
+public def relativeLink (path : Path) (htmlId : Option String := none) : String :=
+  String.join (path.toList.map (· ++ "/")) ++
   (htmlId.map ("#" ++ ·)).getD ""
 
+/--
+Retrieves a string that can be used as an absolute link.
+-/
+public def link (path : Path) (htmlId : Option String := none) : String :=
+  "/" ++ relativeLink path htmlId
```

Comment:
I think we can and should do without this operator. It seems like a footgun - we want documents to be hostable under arbitrary path prefixes.
```suggestion
```